server.api/src/main/java/org/kaazing/gateway/server/ExpiringState.java

```diff
@@ -0,0 +1,60 @@
+/**
+ * Copyright 2007-2016, Kaazing Corporation. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.kaazing.gateway.server;
+
+import java.util.concurrent.TimeUnit;
+
+/**
+ * ExpiringState is accessible from the @javax.security.auth.spi.LoginModule
+ * via the Map<String,?> options, under the name "ExpiringState".
+ *
+ */
+public interface ExpiringState {
```

Comment:
It actually is, but the packages are the same so you wouldn't know unless you expanded the path.  Due to this confusion I've filed this story to combine the modules: https://github.com/kaazing/gateway/issues/700